nntrainer/tensor/weight.h

```diff
@@ -56,7 +56,7 @@ class Weight : public Var_Grad {
    *
    */
   typedef std::tuple<TensorDim, WeightInitializer, WeightRegularizer, float,
-                     bool, std::string>
+                     bool, const std::string>
```

Comment:
is it Dimension of Weight, WeightInitializer, Regularizer, regularizer constant and name? if it is, please add them into brief.

Reply:
Yes, it is. Added in details.